### ISSUE TITLE
Several index loaded in the same time ?

### ISSUE THREAD

**Issue by LaurentClaessens** [spotify/annoy#590]
This question is similar to that one:  https://github.com/spotify/annoy/issues/222

I have already build 2 indexes `ind1.ann` and `ind2.ann`. Both are quite large: ~35Go
Can I load one in one process, and the other in an other process ?

Related question: can I build a third index while the two others are loaded and requested ?

The three indexes have different filenames and are located in distinct directories. (one is even on an other partition if that matters)

I did some tries and I'm getting some strange behaviour : sometimes an index takes up to 400s to be loaded or 10s for one request.

I'm always using `index.load("path/to/file/ind1.ann", prefault=True)`

In short my question is: I wanto to simultaneouslty
- have an index loaded
- load an other (in an other process)
- build a third index in a third process

Is that possible/recommanded ?
